cluster-autoscaler/cloudprovider/cloud_provider.go

```diff
@@ -119,6 +119,9 @@ type NodeGroup interface {
 	// This will be executed only for autoprovisioned node groups, once their size drops to 0.
 	// Implementation optional.
 	Delete() error
+
+	// Autoprovisioned returns true if the node group is autoprovisioned.
```

Comment:
What does it technically mean that a node group is autoprovisioned? Can you be more explicit here?

Reply:
Done.